engine/common/soundlib/ogg_filestream.h

```diff
@@ -0,0 +1,35 @@
+/*
+ogg_filestream.h - helper struct for working with Ogg files
+Copyright (C) 2024 SNMetamorph
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+*/
+
+#ifndef OGG_FILESTREAM_H
+#define OGG_FILESTREAM_H
+
+#include "xash3d_types.h"
+#include <stdint.h>
```

Comment:
As far as I remember, xash3d_types should include stdint.h anyway.

Plus we have pstdint.h for platforms where compiler devs can't be bothered to follow standards. It's included with STDINT_H macro.